### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by ishant005**

### CODE REVIEW

**Review by laxmikanthkonthum — APPROVED** [review]
Looks good to me but just a suggestion to use indentation in your html for better readability

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
+@import url(../styles.css);
+
+.simple-alert-container{
+    display: flex;
+     flex-direction: column;
+     align-items: center;
+    justify-content: center;
```
Comment (rohanmathur91) [inline alert/alert.css]: Indentation is not correct.

File: alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}
+.outline-alert-warning{    
+    border: 2px solid var(--warning-color);
+    color: var(--warning-color);
+}
+.outline-alert-danger{
+    border: 2px solid var(--danger-color);
+    color: var(--danger-color);
+
+}
```
Comment (rohanmathur91) [inline alert/alert.css]: remove blank lines, use any vscode extension like Prettier for formating.

**Review by rohanmathur91 — APPROVED** [review]
Rest looks good.

**Pull request merged**